### ISSUE TITLE
summons command does not validate to summon for a valid review

### ISSUE THREAD

**Issue by shivamb** [etcinit/phabulous#15]
While summoning a reviewer to review for a task it does not pre-check if the reviewer has already reviewed a revision. In either cases if the reviewer has already accepted the revision, rejected or asked for a change the summon command summons the user.

**Comment by etcinit** [comment]
Yes, the current functionality of the `summon` command is very basic. Future versions should consider this and some sort of Slack-to-Phabricator user mapping.

**Comment by etcinit** [comment]
I've begun work on this area. Currently, it seems Phabricator does not return individual reviewer status when querying for revisions. However, it does include the status of the revisions in general (Open, Closed, etc).
